packages/node_modules/@ciscospark/plugin-feature/src/feature.js

```diff
@@ -81,7 +81,8 @@ const Feature = SparkPlugin.extend({
       resource: `features/users/${this.spark.device.userId}/toggles`,
       body: featureList
     })
-      .then((res) => this.spark.device.features[`user`].add(res.body, {merge: true}));
+      .then((res) => res.body.featureToggles.forEach((featTog) => this.spark.device.features.user.add(featTog, {merge: true})))
```

Comment:
There's no guarantee it's a user toggle, right? don't you need to use a property of the `featTog` to determine which collection to add it to?

Also, I *think* there should be a way to bulk add the toggles instead of one at a time; that should reduce the number of change events fired.